src/intl/index.js

```diff
@@ -143,7 +142,7 @@ function removeMessageDescription (messages: { [MessageIdType]: MessageType }):
 
 const defaultMessages: { [MessageIdType]: string } = removeMessageDescription(messages)
 
-function createIdsMap (messages: typeof messages): { [MessageIdType]: MessageIdType } {
+function createIdsMap (messages: { [MessageIdType]: MessageType }): { [MessageIdType]: MessageIdType } {
```

Comment:
IIUIC this change is not required by any updated dependency and we just consider this wording more readable, right?

Reply:
Eslint fails on using symbol before it's declaration here, the `typeof messages` references the local parameter name `messages`. I saw animated discussion regarding eslint vs. flow, it does not play well together.

Semantically, this change is equal, so let's consider it as a readability improvement.

Reply:
It's strange that `typeof messages` worked for me after es-lint update. Let's consider it a readability improvement.